KlaviyoSwift.podspec

```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |s|
   s.swift_version = '5.7'
   s.platform = :ios
   s.ios.deployment_target = '13.0'
-  s.source_files = 'Sources/KlaviyoSwift/**/*.swift'
+  s.source_files = 'Sources/**/**/*.swift'
```

Comment:
Should we do a list here or a regex. I'm wondering if this would capture tests too. Maybe that has a different root directory.

Reply:
yeah tests are a different directory. 

```
klaviyo-swift-sdk/
├── Sources/
│   ├── KlaviyoCore/
│   ├── KlaviyoSwift/
│   ├── KlaviyoSwiftExtension/
│   └── KlaviyoUI/
├── Tests/
│   ├── KlaviyoCoreTests/
│   ├── KlaviyoSwiftTests/
│   ├── KlaviyoSwiftExtensionTests/
│   └── KlaviyoUITests/
└── Package.swift

```